libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Queue/CSFNetwork.m

```diff
@@ -93,7 +93,7 @@ + (instancetype)currentNetwork {
 
 + (instancetype)networkForUserAccount:(SFUserAccount*)account {
     CSFNetwork *instance = nil;
-    if (!account.isTemporaryUser) {
+    if (account!=nil) {
```

Comment:
@qingqingliu 
> Am I missing something?

No, I am. 🙂  I was missing the initial `nil` check.  My argument is pedantic anyway—I'd prefer we didn't rely on `networkForUserAccount:nil` returning a network with an unauthenticated context, in favor of providing an explicit interface like `globalNetwork`, like we do for "app-level" SmartStore class instances.  I think @trooper2013 is working on that approach.